public/index.html

```diff
@@ -1,72 +0,0 @@
-<!DOCTYPE html>
```

Comment:
What purpose was this file serving before out of curiosity?

Reply:
This was the index.html that CRACO (Webapck) was using to bundle with. With Vite this has moved to the `src` directory as required by Vite.